### PR TITLE
Ioda-converters for the [3D]-RTMA AIRCFT data

### PR DESCRIPTION
**Pull request by PraveenKumar-NOAA**
## Description
This PR adds two ioda-converters YAMLs for the [3D]-RTMA aircft data using the following files:

-  testinput/bufr_ncep_rtma_aircft.yaml
-  testinput/bufr_ncep_rtma_aircft_NC004103.yaml
- testinput/rtma_ru.t0000z.aircft.tm00_nc004006.bufr_d
- testinput/rtma_ru.t0000z.aircft.tm00_nc004103.bufr_d
- testoutput/rtma_ru.t0000z.aircft.tm00_nc004006.nc
- testoutput/rtma_ru.t0000z.aircft.tm00_nc004103.nc

## Issue(s) addressed
#1359 #1373 
Resolves #<1359>

## Acceptance Criteria (Definition of Done)

- Good reviews
- Validation of the output results, which are performed in #1373 
- The _ioda-validate.x_ results with the _ObsSpace.yaml_ 

**Final results**:
errors: 0
warnings: 0

## Dependencies
There are no dependencies.

## Impact
None

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have run the unit tests before creating the PR

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -301,6 +301,10 @@ if( iodaconv_bufr_query_ENABLED )
     testinput/bufr_query_fieldname_validation.py
     testinput/bufr_ncep_rtma_mesonet.yaml
     testinput/bufr_ncep_rtma_adpsfc.yaml
+    testinput/rtma_ru.t0000z.aircft.tm00_nc004006.bufr_d
```
Comment (nicholasesposito) [inline test/CMakeLists.txt:304]: Is an rtma aircraft file different than a regular aircraft file?

**Review by nicholasesposito — COMMENTED** [review]
Just one question. Everything looks fine though

File: test/CMakeLists.txt (hunk 1 above)
Comment (PraveenKumar-NOAA) [reply test/CMakeLists.txt:304]: @nicholasesposito thanks, there are separate bufr files for global and regional data. BUFR table is same for both, but size and variables (mnemonics) vary.

Comment (nicholasesposito) [reply test/CMakeLists.txt:304]: Understood. Thanks

**Review by MatthewMorris-NOAA — APPROVED** [review]
Looks good.

**Review by srherbener — APPROVED** [review]

**Review by nicholasesposito — APPROVED** [review]

**Review by PatNichols — APPROVED** [review]

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -1708,6 +1714,24 @@ if(iodaconv_bufr_query_ENABLED)
...
+                            rtma_ru.t0000z.aircft.tm00_nc004006.nc ${IODA_CONV_COMP_TOL_ZERO}
+                    DEPENDS bufr2ioda.x )
+
+  ecbuild_add_test( TARGET  test_iodaconv_bufr_ncep_rtma_aircft_NC004103
+                    TYPE    SCRIPT
+                    COMMAND bash
+                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                            netcdf
+                            "${CMAKE_BINARY_DIR}/bin/bufr2ioda.x testinput/bufr_ncep_rtma_aircft_NC004103.yaml"
+                            rtma_ru.t0000z.aircft.tm00_nc004103.nc ${IODA_CONV_COMP_TOL_ZERO}
```
Comment (BenjaminRuston) [inline test/CMakeLists.txt:1732]: did you want to do a little more renaming to make this a bit more descriptive:
`rmta_ru.20230610T00Z.aircft.tm00_nc004103.nc`

just a preference to have some indication by file name when the data may be valid

**Review by BenjaminRuston — APPROVED** [review]
new ctests pass also tested on an file from RDA:
`gdas.aircft.t00z.20230317.bufr`

and successfully generated IODA file

File: test/CMakeLists.txt (hunk 1 above)
Comment (BenjaminRuston) [reply test/CMakeLists.txt:305]: tested on those two files it worked successfully for the `aircft` file

**Review by BenjaminRuston — APPROVED** [review]
hit wrong button on comment, still working will discuss how to test on full volumes in the future

File: test/CMakeLists.txt (hunk 1 above)
Comment (PraveenKumar-NOAA) [reply test/CMakeLists.txt:305]: > are the regional aircraft files available in NOMADS? I don't even see `aircraft` but see `airsev`
> 
> on the NCAR RDA ,, NOAA provides files like `gdas.aircar.t00z.20230317.bufr` and then the `ds351.0/bufr/2023/gdas.aircft.t00z.20230317.bufr` do you know if the `aircar` are the regional ones?

@BenjaminRuston thanks, I don't know if regional aircraft files are available in NOMADS. But aircar are also regional ones.

File: test/CMakeLists.txt (hunk 2 above)
Comment (PraveenKumar-NOAA) [reply test/CMakeLists.txt:1732]: > did you want to do a little more renaming to make this a bit more descriptive: `rmta_ru.20230610T00Z.aircft.tm00_nc004103.nc`
> 
> just a preference to have some indication by file name when the data may be valid

Comment (PraveenKumar-NOAA) [reply test/CMakeLists.txt:1732]: @BenjaminRuston what you suggest? The file name in tank is - rtma_ru.t00z.aircft.tm00.nc. I added the date and subset name to this file. I think it is ok.

Comment (BenjaminRuston) [reply test/CMakeLists.txt:1732]: think you may have mistyped the tank file I presume is `bufr` was just saying we could break slightly with convention and add more time information to the `t0000z` part instead using `20230610t0000z` 

or something like that.  presume the bufrtanks have directories with the date information.  anyway not critical, and unless Emily or Andrew think it's a fantastic idea we can just proceed

**Pull request merged**